test/sql-tap/map.test.lua

```diff
@@ -991,21 +991,6 @@ test:do_test(
         {abc = 2, [1] = 3}
     })
 
-local remote = require('net.box')
-box.cfg{listen = os.getenv('LISTEN')}
-box.schema.user.grant('guest', 'execute', 'sql')
-local cn = remote.connect(box.cfg.listen)
-test:do_test(
-    "builtins-13.2",
```

Comment:
Similarly to the comment above, I would move `builtins-13.1` as well to keep them together. *Feel free to ignore.*

Reply:
Agree, fixed:

```diff
diff --git a/test/sql-luatest/map_test.lua b/test/sql-luatest/map_test.lua
index b21d065c6b..fefa4e7255 100644
--- a/test/sql-luatest/map_test.lua
+++ b/test/sql-luatest/map_test.lua
@@ -321,6 +321,16 @@ g.test_map_9 = function()
     end)
 end
 
+-- Make sure that MAP values can be used as a bound variable.
+g.test_builtins_13_1 = function()
+    g.server:exec(function()
+        local sql = [[SELECT #a;]]
+        local arg = {{['#a'] = {abc = 2, [1] = 3}}}
+        local res = {abc = 2, [1] = 3}
+        t.assert_equals(box.execute(sql, arg).rows[1][1], res)
+    end)
+end
+
 g.test_builtins_13_2 = function()
     local conn = g.server.net_box
     local ok, res = pcall(conn.execute, conn, [[SELECT #a;]],
diff --git a/test/sql-tap/map.test.lua b/test/sql-tap/map.test.lua
index 1990c1f8ca..c4b268cbbe 100755
--- a/test/sql-tap/map.test.lua
+++ b/test/sql-tap/map.test.lua
@@ -981,16 +981,6 @@ test:do_catchsql_test(
         1, "Failed to execute SQL statement: wrong arguments for function ZEROBLOB()"
     })
 
--- Make sure that MAP values can be used as a bound variable.
-test:do_test(
-    "builtins-13.1",
-    function()
-        local res = box.execute([[SELECT #a;]], {{['#a'] = {abc = 2, [1] = 3}}})
-        return {res.rows[1][1]}
-    end, {
-        {abc = 2, [1] = 3}
-    })
-
 box.execute([[DROP TABLE t1;]])
 box.execute([[DROP TABLE t;]])
 
```